src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -256,11 +256,6 @@ input.inline {
     margin: 0 auto;
 }
 
-.min-height-0.checkbox,
```

Comment:
this CSS is no longer used anywhere

Reply:
Indeed it might be the case, and I might be able to give 10+ more rules that are not used, but next time try to refrain from doing something unrelated to the issue.